.circleci/config.yml

```diff
@@ -1,12 +1,13 @@
-version: 2
-jobs:
-  build:
-    machine: true
-    steps:
-    - checkout
+version: 2.1
+orbs:
+  architect: giantswarm/architect@0.16.0
```

Comment:
Circle setup was very outdated.

Using latest orb means we need replaces for the security issues found by Nancey.